Add a note about the existence of 'next/script' (#27725)
vercel/next.js: docs/api-reference/next/head.md

`title`, `meta` or any other elements (e.g. `script`) need to be contained as **direct** children of the `Head` element,
or wrapped into maximum one level of `<React.Fragment>` or arrays—otherwise the tags won't be correctly picked up on client-side navigations.

> > We recommend using [next/script](https://nextjs.org/docs/basic-features/script) in your component instead of manually creating a `<script>` in `next/head`.

Reviewer: ```suggestion
> We recommend using [next/script](/docs/basic-features/script.md) in your component instead of manually creating a `<script>` in `next/head`.
```

> > Instead of using a script tag within `next/head`, it might make more sense to use `next/script`. The [Next.js Script component](https://nextjs.org/docs/basic-features/script) enables developers to set the loading priority of third-party scripts to save developer time and improve loading performance. 

Reviewer 2: ```suggestion
> We recommend using [next/script](https://nextjs.org/docs/basic-features/script) in your component instead of manually creating a `<script>` in `next/head`.
```
